.github/workflows/bikes.yml

```diff
@@ -49,12 +56,13 @@ jobs:
     - name: Helm Install PR
       run: |
         helm init --upgrade --wait
-        helm upgrade --install --namespace $GITHUB_HEAD_REF bikesharing-$GITHUB_HEAD_REF samples/BikeSharingApp/Bikes/charts/bikes \
+        helm upgrade --install --namespace ${{steps.replace.outputs.valid_child_namespace}} bikesharing-${{steps.replace.outputs.valid_child_namespace}} samples/BikeSharingApp/Bikes/charts/bikes \
           --set image.repository=${{ secrets.CONTAINER_REGISTRY }}/bikes \
           --set image.tag=$GITHUB_SHA \
           --set imagePullSecrets[0].name=${{ secrets.IMAGE_PULL_SECRET }}
  
-    - uses: azure/dev-spaces/actions/add-review-url@Releases/v1              
+    - uses: azure/dev-spaces/actions/add-review-url@Releases/v2
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}  
-        host: ${{ secrets.HOST }}
\ No newline at end of file
+        host: ${{ secrets.HOST }}
+        child-namespace: ${{steps.replace.outputs.valid_child_namespace}}
```

Comment:
In dev spaces, we generally refer to Kubernetes namespaces that have been upgraded to a dev space as a "dev space", or if the context of "dev spaces" is already apparent, just a "space". So here, we would name it "child-space".

Reply:
updated